Tella/Scenes/CommonConnectionReport/Views/ReportMainView.swift

```diff
@@ -0,0 +1,236 @@
+//
+//  ReportMainView.swift
+//  Tella
+//
+//  Created by Dhekra Rouatbi on 1/7/2024.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+struct ReportMainView: View {
+    
+    @ObservedObject var reportMainViewModel: ReportMainViewModel
+    @EnvironmentObject var sheetManager: SheetManager
+    @EnvironmentObject var mainAppModel: MainAppModel
+    
```

Comment:
Here also we have a duplicated mainAppModel so we can remove this` @EnvironmentObject var mainAppModel: MainAppModel`